yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1566,12 +1690,23 @@ To manually modify the schema, please refer: <a class="highlight-link" href="htt
 		Type: SizingNotes,
 		Text: `Import data time estimates exclude redundant indexes since they are automatically removed during export schema phase.`,
 	}
+
+	PARTIAL_MULTI_NODE_ASSESSMENT = NoteInfo{
+		Type: SizingNotes,
```

Comment:
I think its a general note as it will affect all the sections like assessment issues, performance optimization and sizing